.github/workflows/macos-latest.yml

```diff
@@ -0,0 +1,36 @@
+
+name: macos-latest
+
+on:
+  push:
+    branches:
+      - '*'
+    tags-ignore:
+      - '*'
+  pull_request:
+
+jobs:
+  perl:
+
+    runs-on: macOS-latest
+
+    strategy:
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Perl
+        run: |
+          brew install perl
```

Comment:
Is this step necessary, we should already have Perl 5.34.0 as by the documentation, https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11-Readme.md

Reply:
probably - let me see